### PR TITLE
Fix command name for `closereport` on README docs

### PR DESCRIPTION
**Pull request by andrerfneves**
## Notes

Quick copy fix for the command name for `closereport` on README docs. Noticed this while testing v0.2.1-alpha and reading through the new docs.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,4 +119,4 @@ make itest
 Individual itests can also be run using:
 ```
 ./run_itest.sh {test name}
-```
\ No newline at end of file
+```
```
Comment (carlaKC) [inline README.md:122]: nit: unrelated change on this line?

**Review by carlaKC — APPROVED** [review]
Nice catch, thanks for the PR 🎉

File: README.md (hunk 1 above)
Comment (andrerfneves) [reply README.md:122]: Hm this was something GitHub UI added? Let me try and clean it up

Comment (carlaKC) [reply README.md:122]: Could be a ghost-diff, I've seen them before 👻 no need to sink too much time into it!

Comment (andrerfneves) [reply README.md:122]: 👍
